### PR TITLE
Added delete job behavior

### PR DESCRIPTION
**Pull request by codercampos**
## Description
Added the delete job behavior on the Job's detail page. This behavior makes a Soft Delete on the Job registry in the database; which means that adds the DeleteAt date value and sets IsActive to false.

## Visual aids
_No visual aids_

### CODE REVIEW

File: Web/Controllers/JobsController.cs [hunk 1]
```diff
@@ -220,6 +220,27 @@ public JsonResult ToggleHide(string title)
...
         }
 
+        [HttpPost]
+        public IActionResult Delete(int? id)
+        {
+            if (id == null)
+                return RedirectToAction("Index", "Home").WithError("El id de trabajo no es válido.");
+            var job = _jobService.GetById(id.Value);
+            if(job == null)
+                return RedirectToAction("Index", "Home").WithError("El id de trabajo no es válido.");
```
Comment (eatskolnikov) [inline Web/Controllers/JobsController.cs]: El mensaje debería leer "La posición solicitada no existe"

File: Web/Controllers/JobsController.cs [hunk 2]
```diff
@@ -220,6 +220,27 @@ public JsonResult ToggleHide(string title)
...
+            if(result.ExecutedSuccesfully)
+            {
+                // TODO Success message here but after getting the alerts working
+                return RedirectToAction("Index", "Home");
+            }
+
+            // Show the real message internally so We can track it and address it
+            Console.WriteLine(result.Message);
+            var seoid = job.Title.SanitizeUrl().SeoUrl(id.Value);
+            return RedirectToAction("Detail", new { id = seoid }).WithError("Hubo un problema al momento de actualizar este trabajo.");
```
Comment (eatskolnikov) [inline Web/Controllers/JobsController.cs]: "Al momento de eliminar la posición"

File: Data/Repositories/BaseRepository.cs [hunk 3]
```diff
@@ -115,6 +115,7 @@ public virtual void SoftDelete(int id)
                 throw new Exception("This object does not exists");
```
Comment (eatskolnikov) [inline Data/Repositories/BaseRepository.cs]: Cambia estoy ponlo en spanish plz

File: Web/Controllers/JobsController.cs (hunk 1 above)
Comment (codercampos) [reply Web/Controllers/JobsController.cs]: Done

File: Web/Controllers/JobsController.cs (hunk 2 above)
Comment (codercampos) [reply Web/Controllers/JobsController.cs]: Done

File: Data/Repositories/BaseRepository.cs (hunk 3 above)
Comment (codercampos) [reply Data/Repositories/BaseRepository.cs]: Done

**Pull request merged**